src/Veldrid/Vk/VkGraphicsDevice.cs

```diff
@@ -506,6 +506,20 @@ private void CreateInstance(bool debug, VulkanDeviceOptions options)
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
+                // The new MoltenVK now requires the following code
```

Comment:
Maybe just link to https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_portability_enumeration.html